src/Client.js

```diff
@@ -105,4 +105,18 @@ export default class Client {
     }
     return Error('Event not recognized!');
   }
+
+  async once(event, callback, options = {}) {
+    if (Object.keys(this.events).includes(event)) {
+      const routingKey = this.events[event];
+      const exchange = this.api.getExchange(routingKey);
```

Comment:
Woopsies :grimacing: 

```suggestion
      const exchange = api.getExchange(routingKey);
```

Reply:
Done! :smiley: